### ISSUE TITLE
Let each projectile type has its own fire rate

### ISSUE THREAD

**Issue by EvoLandEco** [tresinformal/game#516]
IMO after the implementation of fire rate, each type of projectile should have its own fire rate

**Comment by TheoPannetier** [comment]
I agree! But I also wonder about what the rest of the group think of this design decision.
@richelbilderbeek and @sebmader, this makes me think that we don't have a formal framework to discuss design decisions? We have the [Game Design](https://github.com/tresinformal/game/projects/6) kanban but don't really make use of it so far. Maybe there could be a way to exploit it to collectively discuss/validate/reject design decisions?

I'm thinking something like this:
1. Member has an idea for adding/changing something about game design
2. Member writes an issue about it, briefly exposes the proposition (like this issue) and possibly their motivations for it. Member adds label "Idea" to the issue, then assigns it to column "Ideas" in project "Game design"
3. Member signals the issue on Discord, possibly explains it for 5 min during a session, then the idea is submitted to vote. Issue is moved to "In-discussion".
4. When majority (or consensus?) is reached, the issue moves to "Agreed upon" and someone can then write a test for it and implement the proposition, as usual.

**Comment by richelbilderbeek** [comment]
Nice discussion here! The informal flow is to let newbies implement their things, as those new things are usually good ideas. Sure, there is a risk that it gets removed in the future by voting (which has never happened until now). 

Also, when there is need for voting, I can always let that happen!

**Comment by richelbilderbeek** [comment]
An easy solution: use a parameter for both the firing speeds. In that way, they can easily be set to equal values if desired. `game_options` is the best candidate at the moment, although that class should be split up as well (e.g. into `player_options` that contains all constants for a `player`)

**Comment by sebmader** [comment]
Sounds like a fun idea!
To be honest, at first glance I thought that this is not the time to think about such a detail.
But I agree with the loose flow, so, let's take up ideas when they come to mind :)
If possible, the more senior people can take over the documenting of ideas on the respective board.
Not necessarily, so we vote about it before the idea is further developed, but to kind of keep track of our design decisions.

**Comment by richelbilderbeek** [comment]
What about we vote next Wednesday on this? 

For example, at 18:15, each idea gets 3 mins max, after there will be vote. Team members can vote multiple times. Idea with most votes wins. In case of a draw, we vote with the top ideas only, with 1 vote per person. 

To prepare, people interested in this can inventorize the ideas at 17:15 in a meeting room, so that we'll have some solid options to choose from. That meeting is not about picking the best option (that is what the voting is for), it is about finding clear options to vote on.

Great idea @sebmader @EvoLandEco (and @TheoPannetier , I am unsure how much you want to be in the loop!)?

**Comment by EvoLandEco** [comment]
Or collect a list of ideas and then open a vote for them?

**Comment by richelbilderbeek** [comment]
That would be an idea to vote on. Ideas:

 * Richel (i.e. I would enjoy to present it): let the newbies do their thing, only remove things if majority of team thinks it is a bad idea
 * (proposal) Tianjian (tentative): collect a list of ideas first, vote for them, work only on these
